examples/02_async/tsconfig.json

```diff
@@ -8,6 +8,19 @@
     "skipLibCheck": true,
     "noUncheckedIndexedAccess": true,
     "exactOptionalPropertyTypes": true,
-    "jsx": "react-jsx"
-  }
+    "jsx": "react-jsx",
+    "outDir": "./dist",
+    "tsBuildInfoFile": "./dist/.tsbuildinfo",
+    "composite": true,
+    "types": [
+      // React Server Async Component type
+      "react/experimental"
```

Comment:
Can we put this also in the source code as inline comment? Just a preference, but it looks more explicit.

Reply:
This is remaining. Also in 10_dynamicroute.